liveblog/liveblog
# [LBSD-2776] - New posts are not showing up in Editorial tab

### Purpose

This PR fixes an issue where some posts would not show up on the Editorial tab.

### What has changed
- When a live preview is loaded, an `initialRender` function is called which re-fetches and renders the timeline again before fetching new ones. 

### Steps to test
1. Open the Editor.
2. Make sure for your theme settings the default tab is set to Editorial
3. Add several posts and change their order around.
4. Observe the posts show up as intended on the Live Preview

Resolves: [LBSD-2776](https://sofab.atlassian.net/browse/LBSD-2776)

[LBSD-2776]: https://sofab.atlassian.net/browse/LBSD-2776?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
